packages/@react-spectrum/calendar/docs/Calendar.mdx

```diff
@@ -0,0 +1,161 @@
+import i18nDocs from 'docs:@internationalized/date';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/calendar/package.json';
+
+```jsx import
+import {Calendar} from '@react-spectrum/calendar';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Date and Time
+keywords: []
+after_version: 3.0.0
+---
+
+# Calendar
+
+<PageDescription>{docs.exports.Calendar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Calendar']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<Calendar aria-label="Event date" />
+```
+
+## Value
+
+A `Calendar` has no selection by default. An initial, uncontrolled value can be provided to the `Calendar` using the `defaultValue` prop. Alternatively, a controlled value can be provided using the `value` prop.
+
+Date values are provided using objects in the `@internationalized/date` package. This library handles correct international date manipulation across calendars, time zones, and other localization concerns.
+
+`Calendar` only supports selecting dates, but values with time components are also accepted. By default, `Calendar` will emit <TypeLink links={i18nDocs.links} type={i18nDocs.exports.CalendarDate} /> objects in the `onChange` event, but if a <TypeLink links={i18nDocs.links} type={i18nDocs.exports.CalendarDateTime} /> or <TypeLink links={i18nDocs.links} type={i18nDocs.exports.ZonedDateTime} /> object is passed as the `value` or `defaultValue`, values of that type will be emitted, changing only the date and preserving the time components.
+
+```tsx example
+import {parseDate} from '@internationalized/date';
+
+function Example() {
+  let [value, setValue] = React.useState(parseDate('2020-02-03'));
+
+  return (
+    <Flex gap="size-300" wrap>
+      <Calendar
+        aria-label="Date (uncontrolled)"
+        defaultValue={parseDate('2020-02-03')} />
+      <Calendar
+        aria-label="Date (controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+### International calendars
+
+`Calendar` supports selecting dates in many calendar systems used around the world, including Gregorian, Hebrew, Indian, Islamic, Buddhist, and more. Dates are automatically displayed in the appropriate calendar system for the user's locale. The calendar system can be overridden using the Unicode calendar locale extension, passed to the `Provider` component.
+
+By default, all selected dates are converted to the Gregorian calendar, even if they are displayed to the user in a different calendar system. Since applications usually store dates in the Gregorian calendar system, this means you don't usually need to worry about dealing with other calendar systems in application code. However, if you do provide a `value` or `defaultValue` in a non-Gregorian calendar, the selected dates passed to `onChange` will also be in the same calendar system.
```

Comment:
The problem is that most programmers do not know anything about non-gregorian calendars and may write application logic assuming the date is in the Gregorian calendar (what they tested). Then, when the application is used in a different country with another calendar system, the code will suddenly receive a date in a different calendar system but it will be treated like Gregorian. Therefore, I think it's best to always return dates in whatever calendar system the programmer provided the date in (most likely Gregorian), rather than what the user selected. This prevents bugs, but still allows the programmer control if needed.

Reply:
Not great, but I understand. 👍

Reply:
Also updated the text of this paragraph.